crates/ruff_linter/src/rules/flake8_bugbear/rules/abstract_base_class.rs

```diff
@@ -53,7 +61,7 @@ impl Violation for AbstractBaseClassWithoutAbstractMethod {
     #[derive_message_formats]
     fn message(&self) -> String {
         let AbstractBaseClassWithoutAbstractMethod { name } = self;
-        format!("`{name}` is an abstract base class, but it has no abstract methods")
+        format!("`{name}` is an abstract base class, but it has neither abstract methods nor abstract class variables")
```

Comment:
Similarly we can make this a little more concise:

```diff
- format!("`{name}` is an abstract base class, but it has neither abstract methods nor abstract class variables")
+ format!("`{name}` is an abstract base class, but it has no abstract methods or properties")
```

Reply:
https://github.com/cmp0xff/astral-sh-ruff/commit/680b19969fd234a6cb706bb4e68bdeff103a51b1